docker-compose.yml

```diff
@@ -1,22 +1,49 @@
 version: '3.9'
 services:
   server:
+    env_file:
+      - ./docker/db/.env
+      - ./docker/server/.env
     build:
       context: .
       target: hb_back
     image: hb_backend:latest
     volumes:
       - ./backend:/usr/src/app
+      - ./docker/server/docker-entrypoint-init.d:/docker-entrypoint-init.d
     ports:
       - 8085:8080
-    command: go
+    command: server
+    depends_on:
+      - db
+
```

Comment:
Any particular reason for the empty line?

Reply:
Just for spacing